### PR TITLE
287 Hardcode the DNS addresses in the code and create a flag for handling multiple DNS

### PR DESCRIPTION
**Pull request by thomas-senechal**
Note that since we only have a DNS deployed on TestNet 17, we can't add default DNS for the others Networks.

### CODE REVIEW

**Comment by SlnPons** [comment]
@thomas-senechal I don't see any comment on the PR, could we add the label "action:merge"?

**Comment by thomas-senechal** [comment]
> @thomas-senechal I don't see any comment on the PR, could we add the label "action:merge"?

Oops.
In fact, I just forgot to put that PR in review. 
We now have to wait for 2 reviews before adding the "action:merge" label.

File: int/api/server.go
Comment (gregLibert) [inline int/api/server.go:66]: You should avoid to use the package name in the constant. It's redundant.

File: int/api/websites/all.go
Comment (gregLibert) [inline int/api/websites/all.go:29]: Same here: Get and DNS are useless.

File: pkg/onchain/dns/dns.go
Comment (gregLibert) [inline pkg/onchain/dns/dns.go:20]:
```suggestion
const EnvKey = "THYRA_DNS_ADDRESS"

func Address() string {
	return os.Getenv(EnvKey)
}
```

**Review by gregLibert — CHANGES_REQUESTED** [review]
:no_good: The readability of the code needs to be improved

File: int/api/server.go
Comment (gregLibert) [inline int/api/server.go:52]: What does Ptr mean ?

**Review by gregLibert — APPROVED** [review]
:thinking:

File: int/api/server.go
Comment (thomas-senechal) [reply int/api/server.go]: Fixed!

**Review by B-Naoufal — APPROVED** [review]

File: int/api/server.go
Comment (thomas-senechal) [reply int/api/server.go:57]: Discussed in PM

**Pull request merged**
